db/migrate/20200213205538_add_initial_facets.rb

```diff
@@ -0,0 +1,61 @@
+class AddInitialFacets < Mongoid::Migration
```

Comment:
I think Devon addressed the issues here in his [PR](https://github.com/broadinstitute/single_cell_portal_core/pull/393), so you may want to merge in his changes before merging this as I'm worried it will clobber his changes.

Reply:
Addressed by updating with Devon's recent merge, b5db36c.